qa/workunits/ceph-helpers.sh

```diff
@@ -1443,19 +1443,20 @@ function erasure_code_plugin_exists() {
     local plugin=$1
     local status
     local grepstr
+    local s
```

Comment:
@badone @dachary i realized that `local` is a command by itself. so i should run it before assigning the output to `s`, now the captured return code should be correct.